macos/library/FLEViewController.h

```diff
@@ -71,7 +71,19 @@
 - (BOOL)addPlugin:(nonnull id<FLEPlugin>)plugin;
 
 /**
- * Sends a platform message to the Flutter engine on the given channel.
+ * Adds a responder for keyboard events. Key up and key down events are forwarded to all added responders.
+ */
+- (void)addKeyResponder:(nonnull NSResponder*)responder;
```

Comment:
Per the first part of my comment about this change in the previous round, I don't think arbitrary third-party plugins should be able to be key event handlers, so these should be in the internal header.